ui/shared/components/panel/search/VariantSearchFormPanels.jsx

```diff
@@ -103,6 +104,29 @@ export const ANALYST_PATHOGENICITY_PANEL = pathogenicityPanel(true)
 export const PATHOGENICITY_PANEL = pathogenicityPanel(false)
 
 const ANNOTATION_GROUP_INDEX_MAP = ANNOTATION_GROUPS.reduce((acc, { name }, i) => ({ ...acc, [name]: i }), {})
+const IN_SILICO_FILTER_ROW_CHUNK_SIZE = 3
+
+export const inSilicoFieldLayout = (fieldComponents) => {
```

Comment:
You can fit way more than 3 filters per row. Additionally, I think you need to create a visual distinction between numeric and text filters. Try putting all the numeric filter in one row and all the text filters in another row. If those rows look crowded, you can split them into up to 2 rows for each type